shared_model/bindings/model_query_builder_embind.cpp

```diff
@@ -0,0 +1,68 @@
+#include <emscripten/bind.h>
+#include <emscripten/val.h>
+
+#include "model_proto.hpp"
+#include "model_query_builder.hpp"
+
+using namespace emscripten;
+using namespace shared_model;
+using namespace shared_model::bindings;
+using namespace shared_model::proto;
+
+using namespace shared_model::crypto;
+
+EMSCRIPTEN_BINDINGS(model_query_builder) {
+  register_vector<Hash>("HashVector");
+
+  class_<Hash, base<Blob>>("Hash").constructor<const std::string &>().function(
+      "toString", &Hash::toString);
+
+  // TODO 20.06.18 stinger112: Add underlaying types for commented methods
```

Comment:
It will be implemented in the next iteration of **iroha-lib**, because it's not so trivial and needs additional efforts. It's not possible (and not required) to implement all of the functionality at once,
so we with @laSinteZ decided to do it later.